lib/wakes/location_helper.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+class LocationHelper
```

Comment:
play around with `URI()` and see what you think you can do with that. You may get by just putting a couple of helper methods in the matcher... Specifically

    def path(location_string)
      URI(location_string).path
    end
    
    def host(location_string)
      location_string = "http://#{location_string}" unless location_string.starts_with?('http')
      URI(location_string).host
    end